### PR TITLE
Update rspec to work with view inheritance.

### PR DESCRIPTION
**Pull request by squaresurf**
Why:

* https://github.com/rspec/rspec-rails/issues/396 - TL;DR if you use a
partial that is in the `app/views/application/` folder than rspec will
fail.

This change addresses the need by:

* Add the `application` view folder to the view lookup for rspec.

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -11,6 +11,11 @@
   config.use_transactional_fixtures = true
   config.infer_spec_type_from_file_location!
   config.include FactoryGirl::Syntax::Methods
+
+  # https://github.com/rspec/rspec-rails/issues/396
```
Comment (squaresurf) [inline spec/rails_helper.rb]: I'm going to ignore this as I think it is useful to have this one comment.

**Comment by Mrjaco12** [comment]
:+1:

**Comment by bbrock25** [comment]
:+1:

**Comment by samcdavid** [comment]
Why are we writing specs for views? Are they BDD or integration tests?

**Comment by Mrjaco12** [comment]
IMO its useful to make sure that your views render. That way you catch stupid var typos, or changes in view helper function names. Nothing more detailed than that.

**Comment by samcdavid** [comment]
:+1:

**Pull request merged**